```diff
{% raw %}--- a/src/App.js
+++ b/src/App.js
+++ b/Users/ecclemm/Projects/ericclemmons/docs/js/tutorials/photo-albums-in-react/_steps/src/App.js/p1s5.js
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { Grid } from 'semantic-ui-react';
+import 'semantic-ui-css/semantic.min.css';
+
+import Amplify from 'aws-amplify';
+import awsconfig from './aws-exports';
+
+Amplify.configure(awsconfig);
+
+export default function App(props) {
+  return (
+    <BrowserRouter>
+      <Grid padded>
+        <Grid.Column>...</Grid.Column>
+      </Grid>
+    </BrowserRouter>
+  );
+}{% endraw %}
```